tests/integration/new_relations/test_new_relations.py

```diff
@@ -42,20 +42,21 @@ async def test_deploy_charms(ops_test: OpsTest, application_charm, database_char
                 application_charm,
                 application_name=APPLICATION_APP_NAME,
                 num_units=2,
+                series=CHARM_SERIES,
             ),
             ops_test.model.deploy(
                 database_charm,
                 resources={"patroni": "patroni.tar.gz"},
                 application_name=DATABASE_APP_NAME,
                 num_units=1,
-                trust=True,
```

Comment:
JFMI, why trust is no longer necessary?

Reply:
It was not needed before. Probably it was incorrectly copied from the k8s charm tests.